Dockerfile

```diff
@@ -47,7 +47,7 @@ RUN if [ "$LOCALHOST_DEV" ] ; then \
     echo "LOCALHOST_DEV is not set, building production (Oracle) dependencies" && \
     # Converted to Debian format
     apt-get install --yes alien && \
-    curl -sO https://yum.oracle.com/repo/OracleLinux/OL7/oracle/instantclient/x86_64/getPackage/oracle-instantclient${ORACLE_CLIENT_VERSION}-basiclite-${ORACLE_CLIENT_VERSION_FULL}.x86_64.rpm https://yum.oracle.com/repo/OracleLinux/OL7/oracle/instantclient/x86_64/getPackage/oracle-instantclient${ORACLE_CLIENT_VERSION}-devel-${ORACLE_CLIENT_VERSION_FULL}.x86_64.rpm && \
+    curl -sO https://yum.oracle.com/repo/OracleLinux/OL7/oracle/instantclient/x86_64/getPackage/oracle-instantclient${ORACLE_CLIENT_VERSION}-basiclite-${ORACLE_CLIENT_VERSION_FULL}.x86_64.rpm && \
     alien oracle-instantclient*.rpm && \
     dpkg -i *.deb && rm *.deb *.rpm && \
     pip install cx_Oracle==7.0 \
```

Comment:
Looks like we didn't look at updating `cx_Oracle`. Latest version is 8.3. https://pypi.org/project/cx-Oracle/

Reply:
Good point, I'll see if I can upgrade that version here. 

I agree if I re-did this I probably would either have a separate Dockerfile or just include this in the main file without keeping is separate. Back at one time it was a lot harder to build the Oracle piece but now it's more available and a little easier to install.